Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM jacobat/ruby:2.2.1
```

Comment:
hmm this is unusable for anyone outside of zendesk ... some open-source image would be nice ...